globus_cli/commands/session/update.py

```diff
@@ -18,24 +18,27 @@
 )
 @no_local_server_option
 @click.argument("identities", nargs=-1, required=False)
+@click.option("--domain", multiple=True, help="authenticate with a specific domain")
```

Comment:
The behavior right now is that the user has to have authenticated with any of the listed domains in the current session, which I don't think this conveys.

Maybe also add a note that it can be used in conjunction with `prompt=login` to force re-auth if a matching identity is already in session?

There's also some questions now about what exactly the behavior should be in Auth -- see long thread in #testing this morning and the brief discussion during the GCS sync call earlier today.

Reply:
The CLI always sends `prompt=login`.
The helptext will need to change. It needs to state that it's a `multiple=True` opt and that it's mutex with the other options. I think we should figure out what the interface is before we try to finalize text though.

I thought the desired behavior was now clear -- I'll send email to clear up whatever is left.